CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## [1.12.113](https://github.com/brave/brave-browser/releases/tag/v1.12.113)
+
+ - Added Sync v2. ([#10203](https://github.com/brave/brave-browser/issues/10203))
+ - Enabled the "prefetch-privacy-changes" flag by default under brave://flags. ([#8319](https://github.com/brave/brave-browser/issues/8319))
+ - Added support for state level ads delivery. ([#9200](https://github.com/brave/brave-browser/issues/9200))
+ - Added the date of installation to the stats ping. ([#10061](https://github.com/brave/brave-browser/issues/10061))
+ - Added farbling for WebGL API when "Fingerprinting blocking" is set to "strict". ([#10214](https://github.com/brave/brave-browser/issues/10214))
+ - Updated referrer policy to improve privacy and prevent web compatibility issues. ([#8696](https://github.com/brave/brave-browser/issues/8696))
+ - Updated canvas maximum farbling to match balanced farbling. ([#11067](https://github.com/brave/brave-browser/issues/11067))
+ - Updated pre-populated search engine list. ([#11089](https://github.com/brave/brave-browser/issues/11089))
+ - Improved web compatibility by changing behavior of local and session storage in third-party frames to not throw an exception when storage is blocked. ([#9578](https://github.com/brave/brave-browser/issues/9758))
+ - Reduced size and improved performance of the publisher list for Brave Rewards. ([#10836](https://github.com/brave/brave-browser/issues/10836))
+ - Reduced the frequency at which promotions are fetched for rewards. ([#9513](https://github.com/brave/brave-browser/issues/9513))
+ - Fixed issue where "Bat Ads Service" was running when Brave Ads were not enabled. ([#9196](https://github.com/brave/brave-browser/issues/9196))
+ - Fixed crash with Brave Ads when opening a new tab in certain cases. ([#9393](https://github.com/brave/brave-browser/issues/9393))
+ - Fixed issue where "Bat Ledger Service" was running when Brave Rewards was not enabled. ([#9526](https://github.com/brave/brave-browser/issues/9526))
+ - Fixed file-path for cookies as reported on HackerOne by kanytu. ([#9818](https://github.com/brave/brave-browser/issues/9818))
+ - Fixed "Estimated pending rewards" not being refreshed after claiming an ad grant. ([#10094](https://github.com/brave/brave-browser/issues/10094))
+ - Fixed ads state being removed when Brave Ads are disabled. ([#10097](https://github.com/brave/brave-browser/issues/10097))
+ - Fixed clearing URL bar when in edit mode. ([#10524](https://github.com/brave/brave-browser/issues/10524))
```

Comment:
Maybe adding a bit more context re: what's being cleared. Might be obvious to some folks. Something along the lines of the follownig:

```
 - Fixed URL bar text being cleared when going into edit mode.
```